src/readme.rs

```diff
@@ -0,0 +1,8 @@
+///! The only purpose of this file is to run the README.md as if it was a
+///! doctest. The reason for a standalone file is so that any failures show up
+///! clearly as 'readme' failures, as opposed to tests in lib.rs or some other
+///! file.
+///!
+///! https://blog.guillaume-gomez.fr/articles/2020-03-07+cfg%28doctest%29+is+stable+and+you+should+use+it
+
+doc_comment::doctest!("../README.md");
```

Comment:
I feel like this could be a `tests/readme.rs` and not have to live in the src dir?

Reply:
Would be nice, but that didn't work for me. I assume that's because by default, `cargo test` does not do doctest in the tests/ directory? Maybe there is a flag to enable that, but even so I like that it just works right now.